docs/reference/commandline/daemon.md

```diff
@@ -898,6 +900,8 @@ This is a full example of the allowed configuration options in the file:
 	"cluster-store": "",
 	"cluster-store-opts": [],
 	"cluster-advertise": "",
+	"max-concurrency-download": 3,
+	"max-concurrency-upload": 5,
```

Comment:
Shouldn't these be `max-download-concurrency` and `max-upload-concurrency`?

Do you think it would make sense to allow changing these settings at runtime? I think it would be nice to support that, especially since these are brand new options. It should just be a matter of adding a `SetConcurrency` method to `TransferManager`, and adding `SetConcurrency` methods to  `LayerUploadManager` and `LayerDownloadManager` which call `tm.SetConcurrency`.